packages/components/src/ActionIntercom/Intercom.scss

```
@@ -0,0 +1,14 @@
+.tc-intercom {
+	position: relative;
+
+	&.open:after {
```

could be replaced by 
```suggestion
	&-open:after {
```
with the modification of the class